app/javascript/components/payment-error-button.vue

```diff
@@ -32,24 +39,80 @@
 <script>
 
 import invoiceApi from '../api/invoices';
+import { mapState, mapGetters, mapActions } from 'vuex';
```

Comment:
'mapState' is defined but never used                                   no-unused-vars<br>'mapActions' is defined but never used                                 no-unused-vars<br>Absolute imports should come before relative imports                   import/imports-first